### PR TITLE
Audit logs

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds audit logs docs

### Motivation
Public beta 🥳 

### Preview
https://docs-staging.datadoghq.com/sarina/audit-logs/account_management/audit_logs

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by sarina-dd** [comment]
Hi @sidph! I pulled all of the docs we had in Google Docs into this PR. Feel to review and let me know of any additional changes we should make. The one area that I think could use more focus is the monitor's page for audit logs. I'd love to add more concrete examples, if you have any suggestions.

**Comment by sidph** [comment]
This is great - thank you! Let me go through it and share it for more feedback too. There are some last minute pricing updates so we will need to wait to have them cleared (hopefully in the next week) before merging too. I'll let you posted.

File: content/en/account_management/audit_logs.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+- Access Management: Manage access, such as roles and users
+- Authentication: Successful and failed login attempts
+- Dashboard: Additions, deletions, and changes to dashboard resources
+- Log Management: Additions, deletions, and changes to log management resources
+- Monitor: Additions, deletions, and changes to monitor resources
+- Notebooks: Additions, deletions, and changes to notebook resources
+- Request: All web requests
+- Security Monitoring: Additions, deletions, and changes to security monitoring resources
+
+{{< img src="account_management/audit_logs/event-types.png" alt="Audit Logs event types setup in Datadog" style="width:70%;">}}
```
Comment (andrewkandalaft) [inline content/en/account_management/audit_logs.md:46]: I feel like listing these may not be a good idea given they will change pretty frequently over the first year before stabilizing. Also all these details exist on the settings page itself. It may just be preferable to focus on what event types are here. Thoughts?

File: content/en/account_management/audit_logs.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+### Archiving
+
+Archiving is an optional feature for Audit Logs. You can use archiving to write to an S3 bucket and have your SIEM system read events from it. Archiving is a double-write feature that has variable write times, but on average all events are written to the S3 bucket within 15 minutes of generation.
```
Comment (andrewkandalaft) [inline content/en/account_management/audit_logs.md:50]: It lets you write to three cloud storage systems currently: Amazon S3, Google Cloud Storage, and Azure Storage

File: content/en/account_management/audit_logs.md (hunk 1 above)
Comment (sarina-dd) [reply content/en/account_management/audit_logs.md:46]: Agreed, if these are going to change frequently we should remove and put emphasis on what event types are. I don't have a stable understanding as to what we mean when enabling an event type, etc. So any help here is welcome!

File: content/en/account_management/audit_logs.md (hunk 2 above)
Comment (sarina-dd) [reply content/en/account_management/audit_logs.md:50]: Interesting, will update this. Thanks!

Comment (sarina-dd) [inline content/en/account_management/audit_logs.md:50]:
```suggestion
Archiving is an optional feature for Audit Logs. You can use archiving to write to Amazon S3, Google Cloud Storage, or Azure Storage and have your SIEM system read events from it. Archiving is a double-write feature that has variable write times, but on average all events are written within 15 minutes of generation.
```

**Review by sarina-dd — COMMENTED** [review]
Update archiving services

File: content/en/account_management/audit_logs.md (hunk 1 above)
Comment (sarina-dd) [reply content/en/account_management/audit_logs.md:46]: @sidph Hey there! Do we have a standard definition for what event types are and what enabling in the UI means? I can add here to give more context and remove the existing content.

Comment (andrewkandalaft) [reply content/en/account_management/audit_logs.md:46]: Yeah so event types are a collection of audit events. So for example there's the "Authentication" event type which contains all logs related to authentication. or "Dashboards" event type which contains all the logs related to interacting with the dashboards product. Users can toggle on/off which ones are relevant to them.

File: content/en/account_management/audit_logs.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+To explore an audit log, navigate to the [Audit Logs][1] section, also accessible from your [Organizational Settings][2] in the Datadog app.
+
+{{< img src="account_management/audit_logs/explore-audit-logs.png" alt="Audit Logs in the Organization Settings menu" style="width:50%;">}}
+
+Audit Logs have the same functionality as logs within the Datadog [Logs Explorer][4]:
+
+- Filter by facets to drill down into audit logs by `Status` (`Error`, `Warn`, `Inf`o) or `Method` (`POST`, `GET`, `DELETE`).
+- For compliance audits, filter by `event_name` and select `authentication only` to see compliance-related events.
+- Drill down into related audit logs by selecting a log and navigating to the event attributes tab. Select a specific attribute to filter by or exclude from your search, such as `http.method`, `usr.email`, `client.ip`, etc.
+
```
Comment (andrewkandalaft) [inline content/en/account_management/audit_logs.md:75]: @sidph I wonder if it makes sense to be clear about which fields are not expected to be stable enough to write monitors / scripts on. Particularly the description and the prev/new value facets I'd call out as for human eyes. Thoughts?

File: content/en/account_management/audit_logs.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+## Overview
+
+As an administrator or security team member, you can see who is using Datadog within your organization and the context in which they are using Datadog with [Audit Logs][1]. As an individual, you can see a stream of your own actions, too.
+
+There are two types of events that can occur within an audit log:
+
+1. Request events, which translate all requests made to Datadog’s API into a customer record
+2. Product-specific events, such as:
+    - What has changed within your organization or account. For example, if a user changes a dashboard, you are able to see who made the last change and at what time.
+    - Who accessed and viewed certain information within your environment, for compliance audits.
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:21]:
```suggestion
    - User access of information within your environment. For example, for a compliance audit, you can see who accessed or viewed certain information and at what time.
```

File: content/en/account_management/audit_logs.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    - A breaking event, such as seeing what API calls led up to the event.
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:22]: this makes it seem like "seeing what api calls led up to the event" is an example of a breaking event

```suggestion
    - Breaking events. In these cases, you can see what API calls led up to the event.
```

File: content/en/account_management/audit_logs.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    - An audit trail of information relating to user access. For example, if a user receives a policy denied warning.
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:23]: is this different from what's mentioned in line 21?

File: content/en/account_management/audit_logs.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+
+## Setup
+
+Audit Logs is self-service enabled. To enable, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*. Click the **Enable** button.
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:27]:
```suggestion
To enable Audit Logs, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*. Click the **Enable** button.
```

File: content/en/account_management/audit_logs.md [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+
+{{< img src="account_management/audit_logs/setup.png" alt="Audit Logs setup in Datadog" style="width:100%;">}}
+
+## Configuration
+
+### Event types
+
+Audit Logs is configurable by event type. Enabled events are specific to actions that take place in the Datadog application. The following event types are supported for Audit Logs:
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:35]:
```suggestion
Audit Logs is configurable by event type. Enabled events are specific to actions that take place in Datadog. The following event types are supported for Audit Logs:
```

File: content/en/account_management/audit_logs.md [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+- Monitor: Additions, deletions, and changes to monitor resources
+- Notebooks: Additions, deletions, and changes to notebook resources
+- Request: All web requests
+- Security Monitoring: Additions, deletions, and changes to security monitoring resources
+
+{{< img src="account_management/audit_logs/event-types.png" alt="Audit Logs event types setup in Datadog" style="width:70%;">}}
+
+### Archiving
+
+Archiving is an optional feature for Audit Logs. You can use archiving to write to Amazon S3, Google Cloud Storage, or Azure Storage and have your SIEM system read events from it. Archiving is a double-write feature that has variable write times, but on average all events are written within 15 minutes of generation.
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:50]:
```suggestion
Archiving is an optional feature for Audit Logs. You can use archiving to write to Amazon S3, Google Cloud Storage, or Azure Storage and have your SIEM system read these archived events from your cloud storage provider. Archiving is a double-write feature that has variable write times, but on average all events are written within 15 minutes of generation.
```

Can we explain what we mean by double-write? For instance: `Archiving is a double-write feature (each event is written to both Audit Logs and to cloud storage) that has variable write times, but on average all events are written within 15 minutes of generation.` but like sounding better than that and stuff

File: content/en/account_management/audit_logs.md [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+
+To enable archiving for Audit Logs, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*. Scroll down to Archiving and click the Store logs toggle to enable.
+
+### Retention
+
+Retaining logs is an optional feature for Audit Logs. To enable, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*.  Scroll down to Retention and click the Retain logs toggle to enable.
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:56]:
```suggestion
Retaining logs is an optional feature for Audit Logs. To enable, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*. Scroll down to Retention and click the Retain logs toggle to enable.
```

File: content/en/account_management/audit_logs.md [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+
+The default retention period for an audit log is 7 days. You can set a retention period between 3 and 90 days.
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:58]:
```suggestion
The default retention period for an audit log is seven days. You can set a retention period between three and 90 days.
```

File: content/en/account_management/audit_logs.md [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+
+{{< img src="account_management/audit_logs/retention.png" alt="Audit Logs Retention setup in Datadog" style="width:80%;">}}
+
+Note: Audit Logs are priced as retained logs, and there is not a cost for ingestion or archiving. See the [Log Management pricing page][3] for more information.
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:62]:
```suggestion
Note: Audit Logs are priced as retained logs, and there is no cost for ingestion or archiving. See the [Log Management pricing page][3] for more information.
```

File: content/en/account_management/audit_logs.md [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+
+## Explore Audit Logs
+
+To explore an audit log, navigate to the [Audit Logs][1] section, also accessible from your [Organizational Settings][2] in the Datadog app.
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:66]:
```suggestion
To explore an audit log, navigate to the [Audit Logs][1] section, also accessible from your [Organizational Settings][2] in Datadog.
```

File: content/en/account_management/audit_logs.md [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+
+{{< img src="account_management/audit_logs/explore-audit-logs.png" alt="Audit Logs in the Organization Settings menu" style="width:50%;">}}
+
+Audit Logs have the same functionality as logs within the Datadog [Logs Explorer][4]:
+
+- Filter by facets to drill down into audit logs by `Status` (`Error`, `Warn`, `Inf`o) or `Method` (`POST`, `GET`, `DELETE`).
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:72]:
```suggestion
- Filter by facets to drill down into audit logs by `Status` (`Error`, `Warn`, `Info`) or `Method` (`POST`, `GET`, `DELETE`).
```

File: content/en/account_management/audit_logs.md [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+- For compliance audits, filter by `event_name` and select `authentication only` to see compliance-related events.
+- Drill down into related audit logs by selecting a log and navigating to the event attributes tab. Select a specific attribute to filter by or exclude from your search, such as `http.method`, `usr.email`, `client.ip`, etc.
+
+{{< img src="account_management/audit_logs/attributes.png" alt="Audit Logs in the Organization Settings menu" style="width:50%;">}}
+
+## Create a Monitor
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:78]:
```suggestion
## Create a monitor
```

File: content/en/account_management/audit_logs.md [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+
+To create a monitor on a type of audit log or by specific log attributes, see the [Audit Logs Monitor documentation][5].
+
+## Create a Dashboard
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:82]:
```suggestion
## Create a dashboard
```

File: content/en/account_management/audit_logs.md [hunk 17]
```diff
@@ -0,0 +1,107 @@
...
+
+Give more visual context to your audit logs with Dashboards. To create an Audit Logs dashboard:
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:84]:
```suggestion
Give more visual context to your audit logs with dashboards. To create an Audit Logs dashboard:
```

File: content/en/account_management/audit_logs.md [hunk 18]
```diff
@@ -0,0 +1,107 @@
...
+
+1. Create a [New Dashboard][6] in Datadog.
+2. Select your visualization: an Audit Logs source can be queried for [Top Lists][7], [Time Series][8], and [Log Stream][9].
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:87]:
```suggestion
2. Select your visualization. You can query an Audit Logs source for [top lists][7], [timeseries][8], and [log streams][9].
```

File: content/en/account_management/audit_logs.md [hunk 19]
```diff
@@ -0,0 +1,107 @@
...
+3. [Graph your data][10]: Under edit, select Audit Logs as the data source, and create a query. Audit Logs are filtered by count, and can be grouped by the different facets. Select a facet and limit.
```
Comment (cswatt) [inline content/en/account_management/audit_logs.md:88]:
```suggestion
3. [Graph your data][10]: Under edit, select *Audit Logs* as the data source, and create a query. Audit logs are filtered by count and can be grouped by different facets. Select a facet and limit.
```

File: content/en/monitors/create/types/audit_logs.md [hunk 20]
```diff
@@ -0,0 +1,57 @@
+---
+title: Audit Logs Monitor
+kind: documentation
+description: "alert when a specified type of audit log exceeds a threshold"
```
Comment (cswatt) [inline content/en/monitors/create/types/audit_logs.md:4]:
```suggestion
description: "Alert when a specified type of audit log exceeds a threshold."
```

File: content/en/monitors/create/types/audit_logs.md [hunk 21]
```diff
@@ -0,0 +1,57 @@
...
+  tag: "Documentation"
+  text: "Configure your monitor notifications"
+- link: "/monitors/downtimes/"
+  tag: "Documentation"
+  text: "Schedule a downtime to mute a monitor"
+---
+
+## Overview
+
+Audit Log monitors alert when a specified type of audit log exceeds a user-defined threshold over a given period of time.
```
Comment (cswatt) [inline content/en/monitors/create/types/audit_logs.md:19]:
```suggestion
Audit Logs monitors alert you when a specified type of audit log exceeds a user-defined threshold over a given period of time.
```

File: content/en/monitors/create/types/audit_logs.md [hunk 22]
```diff
@@ -0,0 +1,57 @@
...
+
+## Monitor creation
+
+To create an [Audit Logs monitor][1] in Datadog, use the main navigation: *Monitors --> New Monitor --> Audit Logs*.
+
+### Define the search query
+
+Define a search query for your Audit Logs. Search queries follow the same [search syntax][2] as in the Log Explorer.
+
+For example, if you want to be alerted when an API key is making a certain amount of requests, set `count by` to API Key ID, `@metadata.api_key.id`. You can then group by specific User ID, `@usr.id`, or User email, `@usr.email`, to receive a notification specifying which user is making the request.
```
Comment (cswatt) [inline content/en/monitors/create/types/audit_logs.md:29]:
```suggestion
For example, if you want to be alerted when a specific API key is making a certain number of requests, set `count by` to that API key ID, `@metadata.api_key.id`. You can then group by a specific user ID, `@usr.id`, or user email, `@usr.email`, to receive a notification specifying which user is making the request.
```

File: content/en/monitors/create/types/audit_logs.md [hunk 23]
```diff
@@ -0,0 +1,57 @@
...
+
+### Set alert conditions
+
+Set an alert threshold of greater than a certain value. For example, if you want to set the max amount of API requests to 15, set the alert threshold to `Alert threshold > 15`. Set the warning threshold to any number before 15 to receive a warning prior to a threshold being met.
```
Comment (cswatt) [inline content/en/monitors/create/types/audit_logs.md:33]:
```suggestion
Set an alert threshold for the value you want to be alerted on. For example, if you want to be alerted when the number of API requests reaches 15 or above, set the alert threshold for number of API requests to `Alert threshold > 15`. Set the warning threshold to any number before 15 to receive a warning prior to a threshold being met.
```

File: content/en/monitors/create/types/audit_logs.md [hunk 24]
```diff
@@ -0,0 +1,57 @@
...
+
+You can also choose to never resolve, or to automatically resolve, an event from a triggered state. Set a value between `[Never]` (default) and `After 24 Hours`.
+
+### Say what's happening
+
+Create a notification name. For example, `API requests threshold met for {{@usr.id}}`. You can use [template variables][3] to automatically populate a username, email, etc. in the title to quickly gain insight into which account or user is triggering an alert.
+
+Create a monitor message. This can include the steps required for team members to resolve an incident if it is occurring.
```
Comment (cswatt) [inline content/en/monitors/create/types/audit_logs.md:41]:
```suggestion
Create a monitor message. This can include the steps required for team members to resolve an incident if one is occurring.
```

File: content/en/monitors/create/types/audit_logs.md [hunk 25]
```diff
@@ -0,0 +1,57 @@
...
+
+You can then select a value from `[Never]` to `Every 24 Hours` to renotify if the monitor has not been resolved. You can also set tags and priority to better correlate data in the event of an incident.
```
Comment (cswatt) [inline content/en/monitors/create/types/audit_logs.md:43]:
```suggestion
You can then select a value from `[Never]` to `Every 24 Hours` to renotify you if the monitor has not been resolved. You can also set tags and priority to better correlate data in the event of an incident.
```

File: content/en/monitors/create/types/audit_logs.md [hunk 26]
```diff
@@ -0,0 +1,57 @@
...
+
+### Notify your team
+
+Select services and team members to notify. For example, alert your on-call Compliance team by Pagerduty, or alert your team by Slack or email to begin assessment of the alert.
```
Comment (cswatt) [inline content/en/monitors/create/types/audit_logs.md:47]:
```suggestion
Select services and team members to notify. For example, you can alert your on-call compliance team with PagerDuty, or alert your team by Slack or email to begin assessment of the alert.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
editorial review

File: content/en/account_management/audit_logs.md (hunk 9 above)
Comment (sarina-dd) [reply content/en/account_management/audit_logs.md:50]: +1 here, @sidph would be great to have more context on this.

File: content/en/account_management/audit_logs.md (hunk 6 above)
Comment (sarina-dd) [reply content/en/account_management/audit_logs.md:23]: Good catch, I think this was a subsequent add to the former bullet point. Will combine!

File: content/en/account_management/audit_logs.md [hunk 27]
```diff
@@ -0,0 +1,107 @@
...
+## Overview
+
+As an administrator or security team member, you can use [Audit Logs][1] to see who is using Datadog within your organization and the context in which they are using Datadog. As an individual, you can see a stream of your own actions, too.
+
+There are two types of events that can occur within an audit log:
+
+1. Request events, which translate all requests made to Datadog’s API into customer records.
+2. Product-specific events, such as:
+    - Changes within your organization or account. For example, if a user changes a dashboard, you are able to see who made the last change and at what time.
+    - User access of information within your environment. For example, for a compliance audit, you can see who accessed or viewed certain information and at what time.
```
Comment (sarina-dd) [inline content/en/account_management/audit_logs.md:21]:
```suggestion
    - User access of information within your environment. For example, for a compliance audit, you can see who accessed or viewed certain information and at what time or if a user receives a policy denied warning.
```

File: content/en/account_management/audit_logs.md [hunk 28]
```diff
@@ -0,0 +1,107 @@
...
+    - Breaking events. In these cases, you can see what API calls led up to the event.
+    - An audit trail of information relating to user access. For example, if a user receives a policy denied warning.
```
Comment (sarina-dd) [inline content/en/account_management/audit_logs.md:23]:
```suggestion
```

File: content/en/account_management/audit_logs.md [hunk 29]
```diff
@@ -0,0 +1,106 @@
...
+
+To enable Audit Logs, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*. Click the **Enable** button.
+
+{{< img src="account_management/audit_logs/setup.png" alt="Audit Logs setup in Datadog" style="width:100%;">}}
+
+## Configuration
+
+### Event types
+
+Audit Logs is configurable by event type. Enabled events are specific to actions that take place in Datadog. The following event types are supported for Audit Logs:
```
Comment (sarina-dd) [inline content/en/account_management/audit_logs.md:34]:
```suggestion
Event types are a collection of audit events. For example, the Authentication event type contains all logs related to authentication and the Dashboards event type contains all the logs related to interacting with the dashboards product. To enable an event type, navigate to the *Audit Logs Settings* section of your [Organization Settings][2] and toggle on event types that are relevant to you.
```

File: content/en/account_management/audit_logs.md (hunk 3 above)
Comment (sarina-dd) [reply content/en/account_management/audit_logs.md:65]: +1, definitely would be helpful to make note of any fields that won't be stable for beta.

Comment (sidph) [reply content/en/account_management/audit_logs.md:65]: I'd reword this one to - 
- Filter to drill down into audit logs by Event Names (Dashboards, Monitors, Authentication, etc), Authentication Attributes (Actor, API Key ID, User email, etc), `Status` (`Error`, `Warn`, `Info`), `Method` (`POST`, `GET`, `DELETE`), and other facets
- Drill down into related audit logs by selecting a log and navigating to the event attributes tab. Select a specific attribute to filter by or exclude from your search, such as `http.method`, `usr.email`, `client.ip`, etc.

File: content/en/account_management/audit_logs.md [hunk 30]
```diff
@@ -0,0 +1,97 @@
...
+
+- Filter by facets to drill down into audit logs by `Status` (`Error`, `Warn`, `Info`) or `Method` (`POST`, `GET`, `DELETE`).
+- For compliance audits, filter by `event_name` and select `authentication only` to see compliance-related events.
+- Drill down into related audit logs by selecting a log and navigating to the event attributes tab. Select a specific attribute to filter by or exclude from your search, such as `http.method`, `usr.email`, `client.ip`, etc.
+
+{{< img src="account_management/audit_logs/attributes.png" alt="Audit Logs in the Organization Settings menu" style="width:50%;">}}
+
+## Create a monitor
+
+To create a monitor on a type of audit log or by specific log attributes, see the [Audit Logs Monitor documentation][5].
```
Comment (sidph) [inline content/en/account_management/audit_logs.md:70]: In case we want to add examples - Set a monitor that triggers when user XYZ logs in, or set a monitor for anytime a dashboard is deleted

File: content/en/account_management/audit_logs.md (hunk 9 above)
Comment (sidph) [reply content/en/account_management/audit_logs.md:40]: Reading through this one, it is an internal technical detail. Maybe lets just say - 'on average, all audit events are archived within 15 minutes of generation'

File: content/en/account_management/audit_logs.md [hunk 31]
```diff
@@ -0,0 +1,97 @@
...
+
+## Overview
+
+As an administrator or security team member, you can use [Audit Logs][1] to see who is using Datadog within your organization and the context in which they are using Datadog. As an individual, you can see a stream of your own actions, too.
+
+There are two types of events that can occur within an audit log:
+
+1. Request events, which translate all requests made to Datadog’s API into customer records.
+2. Product-specific events, such as:
+    - Changes within your organization or account. For example, if a user changes a dashboard, you are able to see who made the last change and at what time.
```
Comment (pgkamat) [inline content/en/account_management/audit_logs.md:20]: The most common examples we hear on Log Management are:
1. I want to know when someone changed the retention of an index
2. I want to know who modified the processing/pipelines and when.
3. I want to know who modified an exclusion filters.
These are common billing impact and access impact questions that Ali and I come across most often. They will resonate more than the dashboard  and breaking events example
